### ISSUE TITLE
Allow manual changing a restored TensorFlowEstimator

### ISSUE THREAD

**Issue by mirikle** [tensorflow/tensorflow#2198]
For TensorFlowEstimator models, currently there's no way to change learning rate once the model is saved and restored.

Referencing: /tensorflow/tensorflow/contrib/learn/python/learn/estimators/base.py

**Comment by terrytangyuan** [comment]
Have you tried changing/accessing `estimator.learning_rate` after you restored the model?

**Comment by mirikle** [comment]
Thanks! Yes, I'm able to access the learning rate, but it doesn't seem like I can change the learning rate from tho, since the flow has already been established after calling restore.

**Comment by terrytangyuan** [comment]
Yeah it's currently not possible. I'll submit a fix soon. cc: @ilblackdragon

**Comment by terrytangyuan** [comment]
@mirikle See #2279. Once it's merged, you'll be able to specify e.g. `new_params={'learning_rate': 0.2}`

**Comment by mirikle** [comment]
@terrytangyuan Thank you so much!

**Comment by terrytangyuan** [comment]
@mirikle It requires different mechanism to make it work. I'll try fix it later when I get a chance.

**Comment by terrytangyuan** [comment]
@ilblackdragon Could you re-open this and assign to me? I'll try work on this soon.

**Comment by drop-out** [comment]
I am facing the same problem. Is this solved now?
